stable/kommander/requirements.lock

```diff
@@ -1,10 +1,10 @@
 dependencies:
 - name: kubeaddons-catalog
   repository: https://mesosphere.github.io/charts/staging
-  version: 0.1.4
+  version: 0.1.5
 - name: kommander-cluster-lifecycle
   repository: https://mesosphere.github.io/kommander-cluster-lifecycle/charts
-  version: 0.4.6
+  version: v0.4.6
```

Comment:
out of curiosity, why are you adding this v? is it a version thing in helm or something? 🤔 only observed that v being added in your PRs 🤷‍♂

Reply:
🤔 unless it is the helm version, i cannot tell